app/controllers/people_controller.rb

```diff
@@ -0,0 +1,19 @@
+class PeopleController < ApplicationController
+
+  def index
+    @all_people = Person.all
+    render :index
+  end
+
+  def show
+    @person = Person.find(params[:id])
+    render :show
```

Comment:
That's amazing! That hadn't been mentioned in class, and I didn't spend the time reading the Rails guides on anything I didn't *have* to in order to get working code, but that's great to know going forward.